<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFrmParameterFreeRDP</class>
 <widget class="QWidget" name="CFrmParameterFreeRDP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set FreeRDP parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Sam File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leSamFile"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QCheckBox" name="cbTlsSecurity">
       <property name="text">
        <string>Tls Security</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbRdpSecurity">
       <property name="text">
        <string>Rdp Security</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="cbEnable">
     <property name="text">
      <string>Enable</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="cbNlaSecurity">
       <property name="text">
        <string>Nla Security</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbNlaExtSecurity">
       <property name="text">
        <string>Nla Extend Security</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePort"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="cbAuthentication">
       <property name="text">
        <string>Authentication</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbMayInteract">
       <property name="text">
        <string>May Interact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbMayView">
       <property name="text">
        <string>May View</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
